messenger.rst

```diff
@@ -2512,6 +2512,15 @@ wherever you need a query bus behavior instead of the ``MessageBusInterface``::
         }
     }
 
+You also can also specify stamps when handling a message. For example, you can
```

Comment:
What does `specify` means here? IS it about adding new stamps to the message? What if you add a stamp of the same type that some existing stamp: is it appended or overwritten? Thanks!